### ISSUE TITLE
Accessibility issue with the podcasts player for blinds peoples

### ISSUE THREAD

**Issue by legeekdv** [akhilrex/podgrab#134]
I,
First, I have to say: Finally I found a self hosted podcast player! Since the time that I was looking. However, I have a problem with part of the interface. 

The podcast player cannot be used by blind people. Screen readers do not see the player control buttons. Is it possible to put real buttons, an example of this type of buttons is in Settings, the "Save" button is perfectly accessible.

Thanks in advance, 
Legeekdv

**Comment by akhilrex** [comment]
Hey

Thanks a lot for bringing this to my attention. The player is a third party
library (amplitudejs) so I might be a little restricted but I will
definitely try.

Thanks and Regards

Akhil Gupta

**Comment by legeekdv** [comment]
I,
I have an idea that could work around the problem:
You can put an option in the settings to be able to start or resume playing the podcast as soon as the player page loads

Thank-you for your prompt response,
Legeekdv
